constraint/Gopkg.toml

```diff
@@ -0,0 +1,37 @@
+required = [
```

Comment:
Nit: Can we reduce this to the minimum required dependencies? It looks like this is copied from Gatekeeper. Would be nice to keep the library as lightweight as possible.

Reply:
Required files are from kubebuilder. I can look and see what we may be able to drop as we are only using bits of kubebuilder here.

Reply:
removed some extraneous libs